### ISSUE TITLE
Analytics for PNI (pre-launch list)

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#5530]
DEPENDENT ON SITE BEING LIVE FIRST + discussion with Mavis here #5426
- [x] once site is live, add events + appropriate links and filters to [the 2020 dashboard](https://datastudio.google.com/reporting/3e7cca08-7764-4c6e-835f-6dffeae3df22)
- [x] do QA of the dashboard

**SHARING TO STAFF**
- [x] Add dashboard to [Mana page](https://mana.mozilla.org/wiki/display/FOUNDATION/MoFo+User+Research)
- [x] generate a bit.ly link (maybe mz.la?): after talking to Audrey, a bit.ly seems the best way for this internal use. [bit.ly/pni-dashboard](http://bit.ly/pni-dashboard)
- [ ] pin it to one of the PNI slack channels
- [ ] ask Jen to include the link in the PNI marketing meetings

**References**
Doc with plan + 2019 baseline [here](https://docs.google.com/document/d/1oWwHlyreicHybrXudDocclclWfef0m1hyUsob6FyL8w/edit?usp=sharing)

@mmmavis @sabrinang can you please notify here once site goes live so I can link the final trackers to the dashboard? Thanks!

cc: @kristinashu

**Comment by taisdesouzalessa** [comment]
Connected events related to the product pages. Waiting on homepage to connect the remaining events.

@sabrinang @natalieworth when you have some time can you please look [at the dashboard ](https://datastudio.google.com/reporting/3e7cca08-7764-4c6e-835f-6dffeae3df22) and let me know if it makes sense to you? I added some conversion rates on pages 3 and 4 and I am not super confident I am getting them right, pinged Anil and Lindsay for input. Let me know if you see any issues in the dashboard related to the ways I am doing the calculations there (or anything else).

**Comment by taisdesouzalessa** [comment]
Events added, QA done.
Next steps: "sharing to staff" list.

**Comment by taisdesouzalessa** [comment]
Closing this issue and I'll open a small one for next sprint more as a checklist so we don't forget. Items to be transfereed to the new ticket:  

• pin it to one of the PNI slack channels
• ask Jen to include the link in the PNI marketing meetings
